cmd/extensions/main.go

```diff
@@ -170,6 +173,21 @@ func main() {
 		health = healthcheck.NewHandler()
 	}
 
+	podReady = true
+	health.AddReadinessCheck("agones-extensions", func() error {
+		if !podReady {
+			return errors.New("extensions Pod failed Readiness check, shutting down")
```

Comment:
nit: This error still seems to imply that something failed, vs the reality is that we're failing readiness checks _because_ we're shutting down. Something like "asked to shut down, failing readiness check".